src/backend/eventEmitters/compositeEventPublisher.ts

```diff
@@ -1,11 +0,0 @@
-import { BudgetTrackingEventEmitter, EventPublisher } from './EventEmitter';
-
-export function buildCompositeEventPublisher(eventPublishers: EventPublisher[]): EventPublisher {
-  const compositeEmitter = new BudgetTrackingEventEmitter();
-
-  compositeEmitter.onAny(((eventName, eventData) => {
-    // @ts-ignore
-    eventPublishers.map((eventPublisher) => eventPublisher.emit(eventName, eventData));
-  }));
-  return compositeEmitter;
-}
```

Comment:
This is not a special logic, that should be part of the "eventEmitters" tools. Anyone knows how to write a function like this.

You may remember my conflict with the "EventEmitter", so I removed some codes here, to remove complexity, since all we want from the "backend user" is to give us an object that only implement an interface with `emit`.